parse/ConditionParser1.cpp

```diff
@@ -105,9 +105,9 @@ namespace {
                 ;
 
             owned_by_5
-                =   tok.OwnedBy_
+                =   (tok.OwnedBy_
                 >>  parse::detail::label(Affiliation_token) >> parse::empire_affiliation_type_enum() [ _a = _1 ]
-                >>  parse::detail::label(Empire_token)      >  parse::int_value_ref()
+                >>  parse::detail::label(Empire_token)      ) >  parse::int_value_ref()
```

Comment:
keep left side of `parse::int_value_ref` aligned with the immediately above `parse`